common/src/main/java/com/wynntils/mc/event/PacketEvent.java

```diff
@@ -28,13 +29,15 @@ public T getPacket() {
     }
 
     @Cancelable
+    @EventThread(EventThread.Type.ANY)
     public static class PacketSentEvent<T extends Packet<?>> extends PacketEvent<T> {
         public PacketSentEvent(T packet) {
             super(packet);
         }
     }
 
     @Cancelable
+    @EventThread(EventThread.Type.ANY)
```

Comment:
Shouldn't packet sending/receiving only occur on IO thread?

Reply:
You'd think. :) I have not digged into the details, but basically this mixin is injected deep in networking code, which is apparently used in stuff like scanning servers for the join server list, and whatsnot. We only use it for debugging purposes, so we're fine with that.